src/test/lib/DataTest.cpp

```diff
@@ -382,6 +396,7 @@ QuicTestConnectAndPing(
     PingStats ClientStats(Length, ConnectionCount, TotalStreamCount, FifoScheduling, UnidirectionalStreams, ServerInitiatedStreams, ClientZeroRtt && !ServerRejectZeroRtt);
 
     MsQuicRegistration Registration(NULL, QUIC_EXECUTION_PROFILE_TYPE_MAX_THROUGHPUT, true);
+
```

Comment:
nit: please don't go changing all the newline spaces here and below. Leave as is.